spec/spec_helper.rb

```diff
@@ -99,11 +99,18 @@
     ActionMailer::Base.deliveries = []
   end
 
+  # This is a hack to get around the ssl_required? method in
```

Comment:
That sounds like a good idea. Its pretty bad that we're modifying the way all the tests runs just for those redirect tests, rather than making them the exception.

I'd like to fix that here cause it's been annoying me for a while and feels like something that might be having side effects.